token-metadata/program/src/state.rs

```diff
@@ -0,0 +1,38 @@
+use {
+    borsh::{BorshDeserialize, BorshSerialize},
+    solana_program::pubkey::Pubkey,
+};
+/// prefix used for PDAs to avoid certain collision attacks
+pub const PREFIX: &str = "metadata";
+
+pub const NAME_LENGTH: usize = 32;
+
+pub const SYMBOL_LENGTH: usize = 10;
+
+pub const URI_LENGTH: usize = 200;
```

Comment:
These are all `MAX_xyz_LENGTH` right?

Reply:
Yeah they're all the max lengths - are you suggesting they should have max in front of them?

Reply:
yep, otherwise it's ambiguous.